### DISCUSSION TITLE
What Pin on the M5StickC-Plus to drive the Spectrum Visualizer LED Matrix?

### DISCUSSION THREAD

**Discussion by LarryXO** [PlummersSoftwareLLC/NightDriverStrip#134] [Q&A] [answered]
Can't quite figure out in the documentation how to drive the data pin of the LED matrix...can someone point me where I've overlooked it?
Thanks. (Got it loaded on the M5StickC-Plus, now trying to drive the display). Is it the Grove connector?

**Comment by LarryXO** [comment, marked as answer]
After some experimentation, it is G26. And if I might digress, I would suggest anyone attempting to build this spend the effort to re-solder everything - my panels had solder blobs randomly across the backs. 

Would anyone like the build notes?

**Reply by damopinn** [reply]
The pins are setup in the globals.h file. to help anyone find the for the different projects.
Maybe you could say where you got your panels, I bought some from aliexpress and the quality is fine.
The back of my panels is completely covered in soldermask, except for the connections

**Reply by LarryXO** [reply]
Mine were purchased through Dave's Amazon link. There were random solder blobs of a high-temp solder on the back of two of the three units I bought. Almost as if they were soldered in a real hurry, and some splashed. I will likely get my next set from Aliexpress. 

Thank you for the comment about the globals.h file. I see them, now.
